src/GoogleAdWordsEventForwarder.js

```diff
@@ -28,12 +28,42 @@
         },
         ENHANCED_CONVERSION_DATA = "GoogleAds.ECData";
 
+    // Declares valid Google consent values
+    var googleConsentValues = {
+        // Server Integration uses 'Unspecified' as a value when the setting is 'not set'.
+        // However, this is not used by Google's Web SDK. We are referencing it here as a comment 
+        // as a record of this distinction and for posterity.
+        // 
+        // Unspecified: 'unspecified',
```

Comment:
Can you link to the server docs and web docs to show which consent values are valid?  That would be good for posterity.

Reply:
Good callout